pages/linux/wtype.md

```diff
@@ -0,0 +1,28 @@
+# wtype
+
+> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
+> More information: <https://github.com/atx/wtype>.
```

Comment:
```suggestion
> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
See also: `ydotool`.
> More information: <https://github.com/atx/wtype>.
```
Possibly worth doing a shoutout similar to what we did here https://github.com/tldr-pages/tldr/pull/13991/files